app/components/map.tsx

```diff
@@ -0,0 +1,101 @@
+"use client";
+
+import React, { useRef, useEffect } from "react";
+import mapboxgl, { LngLat, LngLatLike } from "mapbox-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+import { m } from "framer-motion";
+
+const lookupCoordinates = {
+  geometry: {
+    type: "Point",
+    coordinates: [-122.463733, 37.777448],
+  },
+};
+
+const softStories = [
+  { lng: -122.424145, lat: 37.80379 },
+  { lng: -122.433985, lat: 37.7751 },
+  { lng: -122.40082, lat: 37.76169 },
+  { lng: -122.42539, lat: 37.7195 },
+  { lng: -122.42698, lat: 37.7616 },
+];
+
+/*
+//lng: number = -122.437, lat: number = 37.75
+*/
+
+//const Map = ({ lng = -122.437, lat = 37.75 }: { lng: number; lat: number }) => {
```

Comment:
The same here, if we do not need this, maybe delete it? :)

Reply:
cleaned up!